spec/decorators/offence_decorator_spec.rb

```diff
@@ -185,5 +187,44 @@
       it { expect { call }.not_to raise_error }
       it { is_expected.to eql('mode of trial reason is a string') }
     end
+
+    context 'when the mode of trial reason code means the description should be hidden' do
+      before do
+        allow(offence).to receive(:mode_of_trial_reasons).and_return(mot_reason_ostruct_collection)
+      end
+
+      [1, 2, 6].each do |code|
+        context "when exactly one reason exists, with code #{code}" do
+          let(:mode_of_trial_reason_array) do
+            [{ code: code,
+               description: 'Defendant elects trial by jury' }]
+          end
+
+          it { is_expected.to eql('') }
+        end
+      end
+
+      context 'when more than one reason exists, one of which should be hidden' do
```

Comment:
If I understand the codes correctly then this description and the next, on line 218, are the wrong way round.

Reply:
yes, the context descriptions look like they need swapping around with line `218`

Reply:
👍 have amended/swapped the context descriptions